mods/ts/rules/nod-support.yaml

```diff
@@ -113,7 +113,7 @@ NASAM:
 		Cost: 600
 	Tooltip:
 		Name: S.A.M. Site
-		Description: Nod Anti-Air base defense. \nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
+		Description: Nod Anti-Air base defense.\nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
```

Comment:
"Aircraft" in all 3 Nod defenses is needlessly capitalized IMO.
Also no need for "all" here.